tandemhunter-main/test/test_TandemHunter.py

```diff
@@ -0,0 +1,252 @@
+# Unit tests for Tandem Hunter
+# Author: Helena L. Spiewak
+# Date created: 2021-04-07
+# Date modified:
+
```

Comment:
Longer term optimising: Some tests could be handled with pytest and mocks instead, to emulate files without needing to upload real test files.